lib/c0/c0_kvmultiset.c

```diff
@@ -1052,7 +1050,8 @@ c0kvms_init(void)
 
     assert(c0kvms_cache == NULL);
 
-    c0kvms_cache = kmem_cache_create("c0kvms", sizeof(**kvmsv), alignof(**kvmsv), SLAB_PACKED, NULL);
+    c0kvms_cache = kmem_cache_create("c0kvms", sizeof(**kvmsv), __alignof__(**kvmsv),
```

Comment:
What is `__alignof__` compared to `_Alignof`?

Reply:
It's complicated, but for starters unlike sizeof the new _Alignof requires a fully specified type (e.g., struct foo) and wont allow and expression (e.g.  *fooptr).  Then, apparently, the former returns the "preferred" alignment for the architecture, while the latter returns the minimum alignment.  Shouldnt' be any difference on amd64, but I am curious about s390x.  Apparently there are architectures that don't require much in the way of alignment from an addressing perspective, but will perform better depending upon alignment.  So __alignof__ is the better choice.  I am certain the C committee knows way more than me about pretty much everything, but I am rather baffled by all this, would like to see their rationale..